xmtp_mls/src/builder.rs

```diff
@@ -144,10 +145,6 @@ impl<ApiClient, Db> ClientBuilder<ApiClient, Db> {
             disable_local_telemetry: disable_events,
```

Comment:
I don't see this exposed out through the FFI Bindings? How does this ever get set to true?